papis/commands/update.py

```diff
@@ -66,17 +346,21 @@ def run(document: papis.document.Document,
 
     if not folder or not info:
         from papis.exceptions import DocumentFolderNotFound
+
         raise DocumentFolderNotFound(papis.document.describe(document))
 
     from papis.api import save_doc
+
+    document.clear()
```

Comment:
? This feels odd.. why do you need to clear all the document keys?

Reply:
I think I added this because of deletions. If we only do `update()` then keys that have been deleted will not have any effect in the resulting document.

Reply:
That's a good point. I'm not sure what the semantics of this function are supposed to be? The reason I thought it was weird is because doing
```
papis.commands.update.run(doc, {"year": 2023})
```
will just completely clear the document, i.e. it does `doc = data` instead of any form of update. Maybe we don't want to call this in the `cli`?

Reply:
In the `cli` function, it's calling `run` like
```
    for document, data in processed_documents:
        run(document, data=data, git=git)
```
and we'll need another function or just write out the whole thing here? e.g. replace by
```
    for document, data in processed_documents:
		document.clear()
		document.update(data)
		if git:
			...
```